2021-08-30 05:39:50,169 INFO    MainThread:14980 [wandb_setup.py:_flush():69] setting env: {}
2021-08-30 05:39:50,169 INFO    MainThread:14980 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-30 05:39:50,169 INFO    MainThread:14980 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-30 05:39:50,169 INFO    MainThread:14980 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-34/Base/wandb/run-20210830_053950-355ufwao/logs/debug.log
2021-08-30 05:39:50,169 INFO    MainThread:14980 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-34/Base/wandb/run-20210830_053950-355ufwao/logs/debug-internal.log
2021-08-30 05:39:50,170 INFO    MainThread:14980 [wandb_init.py:init():375] calling init triggers
2021-08-30 05:39:50,170 INFO    MainThread:14980 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 1, 'batch_size': 64, 'learning_rate': 0.001, 'val_split': 0.2, 'Schedular': 'CosineAnnealingLR', 'Augmentation': 'BaseAugmentation'}
2021-08-30 05:39:50,170 INFO    MainThread:14980 [wandb_init.py:init():424] starting backend
2021-08-30 05:39:50,170 INFO    MainThread:14980 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-30 05:39:50,197 INFO    MainThread:14980 [backend.py:ensure_launched():135] starting backend process...
2021-08-30 05:39:50,224 INFO    MainThread:14980 [backend.py:ensure_launched():139] started backend process with pid: 15024
2021-08-30 05:39:50,225 INFO    MainThread:14980 [wandb_init.py:init():429] backend started and connected
2021-08-30 05:39:50,228 INFO    MainThread:14980 [wandb_init.py:init():477] updated telemetry
2021-08-30 05:39:50,229 INFO    MainThread:14980 [wandb_init.py:init():500] communicating current version
2021-08-30 05:39:51,035 INFO    MainThread:14980 [wandb_init.py:init():505] got version response 
2021-08-30 05:39:51,035 INFO    MainThread:14980 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-30 05:39:51,280 INFO    MainThread:14980 [wandb_init.py:init():540] starting run threads in backend
2021-08-30 05:39:55,857 INFO    MainThread:14980 [wandb_run.py:_console_start():1601] atexit reg
2021-08-30 05:39:55,858 INFO    MainThread:14980 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.REDIRECT
2021-08-30 05:39:55,858 INFO    MainThread:14980 [wandb_run.py:_redirect():1480] Redirecting console.
2021-08-30 05:39:55,861 INFO    MainThread:14980 [wandb_run.py:_redirect():1536] Redirects installed.
2021-08-30 05:39:55,861 INFO    MainThread:14980 [wandb_init.py:init():565] run started, returning control to user process
2021-08-30 05:39:55,862 INFO    MainThread:14980 [wandb_watch.py:watch():43] Watching
2021-08-30 05:39:58,282 INFO    MainThread:14980 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 1
2021-08-30 05:39:58,283 INFO    MainThread:14980 [wandb_run.py:_restore():1543] restore
2021-08-30 05:40:00,373 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 752
}

2021-08-30 05:40:00,651 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 752
}

2021-08-30 05:40:01,293 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 9424
}

2021-08-30 05:40:01,396 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 10298
}

2021-08-30 05:40:01,498 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 10298
}

2021-08-30 05:40:01,600 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 752
  total_bytes: 10298
}

2021-08-30 05:40:01,703 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:01,805 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:01,908 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:02,010 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:02,113 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:02,331 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:02,433 INFO    MainThread:14980 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10298
  total_bytes: 10298
}

2021-08-30 05:40:03,637 INFO    MainThread:14980 [wandb_run.py:_show_files():1915] logging synced files
